package.json

```diff
@@ -50,7 +50,7 @@
     "glob": "^7.1.1",
     "ip": "^1.1.5",
     "load-json-file": "^2.0.0",
-    "webdriverio": "^4.8.0",
+    "webdriverio": "4.8.0",
```

Comment:
What's the logic around needing to lock this version down?

Reply:
The latest release has a defect which breaks the visual regression testing service we're using. Theres a defect logged and its being looked into

Reply:
Ah ok, that makes sense.